ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -1627,3 +1627,17 @@ def igesin(self, fname="", ext="", **kwargs):
             out = super().igesin(basename, **kwargs)
 
         return out
+
+    @wraps(_MapdlCore.cmatrix)
+    def cmatrix(self, symfac="", condname="", numcond="", grndkey="",
+                capname="", **kwargs):
+        """Wrap the CMATRIX command to update correctly the `last_response` command. """ 
+
+        """The CMATRIX command needs to run in non-interactive mode"""
```

Comment:
```suggestion
        # The CMATRIX command needs to run in non-interactive mode
```

Don't use triple quote when commenting code.  Triple code is used only in the docstring or long messages (see top of file).

Reply:
Fixed.